lib/std/unicode.zig

```diff
@@ -631,3 +631,32 @@ test "utf8ToUtf16LeWithNull" {
         testing.expect(utf16[2] == 0);
     }
 }
+
+pub fn utf16leStringLiteral(comptime str: [*:0]const u8) [*c]const c_ushort {
+    comptime {
+        var len: usize = 0;
+        while (str[len] != 0):(len += 1){}
```

Comment:
Use `mem.len` here

Reply:
Since this function is supposed to take in a string literal, shouldn't the type of fn argument be: `comptime str: [:0]const u8` and then `str.len` can be used

Reply:
Yep, I think that's the syntax I was looking for. Should change to:

```zig
pub fn utf8ToUtf16LeStringLiteral(comptime str: [:0]const u8) [*c]const c_ushort {
    comptime {
        var dest: [str.len * 2]u16 = undefined;
        var dest_len = utf8ToUtf16Le(&dest, str[0..]) catch |err| @compileError(err);
```